src/components/Form/Form.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+
+import classNames from 'classnames';
+
+export interface FormGroupProps {
+	type?: 'standard' | 'horizontal' | 'inline';
+	children: ReactNode;
+}
+
+export const Form: FunctionComponent<FormGroupProps> = ({
```

Comment:
moet deze component niet zijn eigen state bijhouden?

Reply:
Daarvoor hebben we een FormRenderer nodig die adhv. config deze gaat opvullen. Volgens mij is dat geen presentational component die hier thuishoort maar een deeltje dat in de client geïmplementeerd moet worden.